pkgs/misc/themes/sddm/chili/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, qtbase, qtquickcontrols, qtgraphicaleffects }:
+
+/*
+  This derivation installs the chili theme. Use
+    `services.xserverdisplayManager.sddm.theme = "chili";`
+  to enable the theme.
```

Comment:
You also need to install the package.

I don't think it's that useful to put installation instructions in comments in the `.nix` files. Probably the right place would be a section on sddm in the manual, or an extension to the module option doc for the `sddm.theme` option.

Reply:
TBH it would probably be better to teach the sddm module a `themePackages` option rather than relying on installing it system-wide, but that's another kettle of fish.

Reply:
Certainly. There is an example for lightdm greeters in nixpkgs.